src/main.rs

```diff
@@ -57,28 +57,31 @@ fn main() {
         error("Error: could not get current directory.");
     }
 
-    // Get the root directory for the search
-    let mut root_dir_buf = match matches.value_of("path") {
-        Some(path) => PathBuf::from(path),
-        None => current_dir.to_path_buf(),
+    //Get one or more root directories to search.
+    let mut dir_vec: Vec<_> = match matches.values_of("path") {
+        Some(paths) => {
+            paths
+                .map(|path| {
+                    let path_buffer = PathBuf::from(path);
+                    if !fshelper::is_dir(&path_buffer) {
+                        error(&format!(
```

Comment:
There was a request somewhere in the open issue that we also allow for files to be specified here (that would be ignored), but I think it's a good idea to explicitely show errors here. The original problem could be solved by using `drop*/` globs instead of `drop*`, I guess. So :+1:

Reply:
Accidentally killing two birds with one stone, glad I could help 😄

Reply:
> The original problem could be solved by using drop*/ globs 

Thanks for this trick! If I come up with a use case for accepting also files, I will open a new issue.